esp/src/src/nls/es/hpcc.ts

```diff
@@ -785,6 +825,9 @@ export = {
     ThorNetworkAddress: "Dirección de red de Thor",
     ThorProcess: "Proceso de Thor",
     Time: "Tiempo",
+    TimeMaxTotalExecuteMinutes: "Tiempo máximo total de minutos de ejecución",
+    TimeMeanTotalExecuteMinutes: "Tiempo promedio total de minutos de ejecución",
+    TimeMinTotalExecuteMinutes: "Tiempo Mínimo Total Ejecutar Minutos",
```

Comment:
just pointing out that Minimo is capitalized here, but maximo and promedio isn't in the previous 2 translations.